tests/test_management.py

```diff
@@ -56,7 +56,7 @@ def setUp(self):
 
     def test_enterprise_customer_not_found(self):
         faker = FakerFactory.create()
-        invalid_customer_id = faker.uuid4()
+        invalid_customer_id = faker.uuid4()  # pylint: disable=no-member
```

Comment:
@snowcrshd Almost definitely false positive (and it didn't complain about this before). New version of pylint being more rigorous than he old one? Or maybe a plugin is missing?